ui/tests/acceptance/dashboard-test.js

```diff
@@ -156,6 +170,8 @@ module('Acceptance | landing page dashboard', function (hooks) {
       }));
       await authPage.login();
       await visit('/vault/dashboard');
+      await click('[data-test-do-not-show-me-again]');
```

Comment:
If the new login flow is going to always direct to dashboard, and the modal is also always going to pop up, this popup is going to be a problem for every acceptance test unless we disable in the test environment (only show if `!Ember.testing`). 
I'm also a little concerned that this is extremely obtrusive for a typical workflow, especially considering we don't have metrics on how often Vault UI users clear their browser storage. I know we've already adjusted this behavior a couple times but it might be worth bringing up these concerns to Product & Design to double check?